CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-- Add documentation for 'mode' field in FileInfo.
```

Comment:
This was a docs update that doesn't affect the version number, so I think it shouldn't be in the changelog.